CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

alissonsilva-cronapp opened the pull request:
https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

alissonsilva-cronapp committed:
CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

drigores reviewed (changes requested):
Existe risco de quebrar caso não tenha o datasource? deveria ter validação?

alissonsilva-cronapp committed:
CRONAPP-3307 - Caixa de Seleção Multipla não atualiza Fonte

https://cronapp.atlassian.net/browse/CRONAPP-3307

## Problema:
Após inserir um registro e solicitar atualização da fonte de dados pelo servidor, o item adicionado não é exibido no componente da caixa de seleção multipla.
## Solução:
Inserido em options a opção de checar os dados da fonte de dados que popula o componente da caixa de seleção multipla.

alissonsilva-cronapp commented:
Adicionado validação
